src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/UserTransformerTest.kt

```diff
@@ -195,39 +195,6 @@ class UserTransformerTest {
       )
     }
 
-    @ParameterizedTest
-    @EnumSource(
-      value = UserRole::class,
-      names = [
-        "CAS1_ADMIN",
-        "CAS1_JANITOR",
-        "CAS1_APPEALS_MANAGER",
-        "CAS1_ASSESSOR",
-        "CAS1_MATCHER",
-        "CAS1_CRU_MEMBER",
-        "CAS1_CRU_MEMBER_FIND_AND_BOOK_BETA",
-        "CAS1_FUTURE_MANAGER",
-        "CAS1_WORKFLOW_MANAGER",
-        "CAS1_REPORT_VIEWER",
-        "CAS1_USER_MANAGER",
-      ],
-      mode = EnumSource.Mode.EXCLUDE,
-    )
-    fun `transformJpaToApi CAS1 should return no permissions for Approved Premises roles which have no permissions defined`(role: UserRole) {
-      val user = buildUserEntity(
-        role = role,
-        apArea = ApAreaEntityFactory().produce(),
-        cruManagementArea = Cas1CruManagementAreaEntityFactory().produce(),
-      )
-
-      every { apAreaTransformer.transformJpaToApi(any()) } returns apArea
-
-      val result =
-        userTransformer.transformJpaToApi(user, approvedPremises) as ApprovedPremisesUser
-
-      assertThat(result.permissions).isEmpty()
-    }
-
```

Comment:
why no need for this anymore?

Reply:
I don't think the test is really adding any value and we're constantly having to update it when we change permissions on roles